src/cmd_line/commandLine.ts

```diff
@@ -88,7 +88,8 @@ class CommandLine {
     } catch (e) {
       if (e instanceof VimError) {
         if (e.code === ErrorCode.E492 && configuration.enableNeovim) {
-          await vimState.nvim.run(vimState, command);
+          const statusBarText = await vimState.nvim.run(vimState, command);
+          StatusBar.setText(vimState, statusBarText);
```

Comment:
Should be same statement as line 94 (isError = true); only difference would be the text that is displayed.